### ISSUE TITLE
load_settings error

### ISSUE THREAD

**Issue by jrr1984** [MSLNZ/msl-equipment#3]
Hi, sorry to bother again.. im running into troubles when running:

motor.load_settings()

i get the following error, thanks in advance.
---------------------------------------------------------------------------
ThorlabsError                             Traceback (most recent call last)
<ipython-input-51-161c3f18ce37> in <module>()
      1 # load the configuration settings (so that we can use the get_real_value_from_device_unit() method)
----> 2 motory.load_settings()

C:\Users\USUARIO\Anaconda3\envs\motorcitos\lib\site-packages\msl\equipment\resources\thorlabs\kinesis\kcube_stepper_motor.py in load_settings(self)
   1056             If not successful.
   1057         """
-> 1058         self.sdk.SCC_LoadSettings(self._serial)
   1059 
   1060     def load_named_settings(self, settings_name):

C:\Users\USUARIO\Anaconda3\envs\motorcitos\lib\site-packages\msl\equipment\resources\thorlabs\kinesis\motion_control.py in errcheck_true(self, result, func, args)
    136         if not result:
    137             msg = '{}.{}{} -> {}'.format(self.__class__.__name__, func.__name__, args, result)
--> 138             self.raise_exception('The function did not return True.\n' + msg)
    139         return result
    140 

C:\Users\USUARIO\Anaconda3\envs\motorcitos\lib\site-packages\msl\equipment\connection.py in raise_exception(self, msg)
     86         """
     87         self.log_error('{!r} {}'.format(self, msg))
---> 88         raise self._exception_handler('{!r}\n{}'.format(self, msg))
     89 
     90     @staticmethod

ThorlabsError: KCubeStepperMotor<Thorlabs|KST101|26001718 at SDK::Thorlabs.MotionControl.KCube.StepperMotor.dll>
The function did not return True.
KCubeStepperMotor.SCC_LoadSettings(b'26001718',) -> False

**Comment by jborbely** [comment]
I have contacted Thorlabs about issues calling `XXX_LoadSettings` and `XXX_LoadNamedSettings` functions for many of their devices (not just the KST101) about 1 year ago and the Thorlabs representative told me not to use these functions because they were unstable. It seems as though the Thorlabs developers still haven't fixed their bugs.

However, the `motor.load_settings()` and `motor.load_named_settings(name)` functions do work for me with my KST101. I have used this function with v1.11.2, v1.14.10, v1.14.12, v1.14.15 and v1.14.17 of Kinesis x64.

My suggestions:

1. Upgrade to v1.14.17 of Kinesis and hope for the best. I use the 64-bit versions of Kinesis so I don't know what the 32-bit versions are like.
2. Try calling `motor.load_named_settings(name)`. To determine what `name` should be open the _ThorlabsDefaultSettings.xml_ file (for me it is located in C:\Program Files\Thorlabs\Kinesis) and find the appropriate `name` in one of the `<DeviceSettingsDefinition Name="...">` XML elements for the stage that is connected to your KST101. For example, my KST101 has a `ZFS13B` stage connected to it and so from the _ThorlabsDefaultSettings.xml_ file there is a `<DeviceSettingsDefinition Name="ZFS13(B)">` XML element so I would use `motor.load_named_settings('ZFS13(B)')`. **I just fixed a bug in my code that calls this function so you will need to re-install `msl-equipment` if you want to try this function.** _You may still get a Kinesis error, but I cannot control that. It's a bug for the Thorlabs developers to fix._
3. Call `motor.set_motor_params(steps_per_rev, gear_box_ratio, pitch)`, see [here](https://github.com/MSLNZ/msl-equipment/issues/1#issuecomment-399842353) for the reasons why. To determine what the input parameters are for the stage that is connected to your KST101 you would find the XML element from Suggestion 2 above. For example, for my `ZFS13B`
```xml
<DeviceSettingsDefinition Name="ZFS13(B)">
    ...
        <Pitch>0.99</Pitch>
        <StepsPerRev>24</StepsPerRev>
        <GearboxRatio>44</GearboxRatio>
    ...
```
so, I'd use  `motor.set_motor_params(24, 44, 0.99)`.

**In v1.14.17 of Kinesis I have a lot of problems with my devices not showing up in the Kinesis GUI. So, keep previous versions of the Kinesis installer to downgrade to previous versions.**
